_pages/pages/documentation/renaming-site-repository.md

```diff
@@ -6,14 +6,17 @@ navigation: pages
 sidenav: pages-documentation
 ---
 
-## Updating your Pages site after renaming your repository
+## Renaming your Pages site repository
 
-Renaming your site repository will interrupt a site's ability to build.  In order to resume site building, it will require you to add a new site to Pages with the new repository url.  (Note: Please be aware that if your site is already live and its repository has been renamed, it will be frozen until a new site is added to Pages at the new repository and a security scan is successfully performed on your new site.)
+Renaming your site's Github repository will interrupt its ability to build. In order to resume site building, it will require you to notify our support team <a href="mailto:pages-support@cloud.gov">pages-support@cloud.gov</a> about your change in repository organizations and/or name. (Note: Please be aware that if your site is already live and its repository has been renamed, it will be frozen until the Pages team can update your site's Github repository.)
```

Comment:
```suggestion
Renaming your site's GitHub repository will interrupt its ability to build. In order to resume site building, it will require you to notify our support team <a href="mailto:pages-support@cloud.gov">pages-support@cloud.gov</a> about your change in repository organizations and/or name. (Note: Please be aware that if your site is already live and its repository has been renamed, it will be frozen until the Pages team can update your site's Github repository.)
```